user-service/routes/user-routes.js

```diff
@@ -1,17 +1,19 @@
 import express from 'express';
+import { getUser } from '../controller/user-controller.js';
 import { createUser } from '../controller/user-controller.js';
 import { loginUser } from '../controller/user-controller.js';
 import { logoutUser } from '../controller/user-controller.js';
 import { changePassword } from '../controller/user-controller.js';
 import { deleteUser } from '../controller/user-controller.js';
+import { authenticateJwt } from '../middleware/auth.js'
 
 const router = express.Router();
 
-router.get('/', (_, res) => res.send('Hello World from user'))
+router.get('/', authenticateJwt, getUser)
 router.post('/signup', createUser)
 router.post('/login', loginUser)
-router.post('/logout', logoutUser)
-router.put('/change_password', changePassword)
-router.delete('/delete_account', deleteUser)
+router.get('/logout', authenticateJwt, logoutUser)
+router.put('/change_password', authenticateJwt, changePassword)
+router.post('/delete_account', authenticateJwt, deleteUser)
 
 export { router };
```

Comment:
I think there's no need to have the curly brackets around router. Personally I do `export default router`.

Reply:
Solved